src/components/patients/index.tsx

```diff
@@ -2,19 +2,46 @@ import React from "react";
 import Header from "components/header";
 import PatientTable from "components/patients/patientTable";
 import SearchTool from "components/patients/searchTool";
+import { getPatients, getPatientsPerQuery } from "services/api";
+import { PatientBundle } from "types";
 import { Card, Elevation } from "@blueprintjs/core";
+
 import "./style.css";
 
+interface Props {
+  onSearch: Function;
+  searchItem: any;
+}
+
 const Patients = () => {
+  const [patientBundle, setPatientBundle] = React.useState({} as PatientBundle);
+
+  const handleSearch = async (searchNameParams: any, searchParams: any) => {
+    const bundle: PatientBundle = await getPatientsPerQuery(
+      searchNameParams,
+      searchParams
+    );
+    setPatientBundle(bundle);
+  };
+
+  React.useEffect(() => {
+    const fetchPatients = async () => {
+      const bundle: PatientBundle = await getPatients();
```

Comment:
Does this get all the patients in the API?
Why is it in a useEffect if the same thing is done each time?

Reply:
When I remove the useEffect, the getPatients() function is called multiple times.
```
  React.useEffect(() => {
    const fetchPatients = async () => {
      const bundle: PatientBundle = await getPatients();
      setPatientBundle(bundle);
    };
    fetchPatients();
  }, []);
```
With the useEffect updated on nothing ([] line 33), the function getPatients is only called one time